src/front/glsl/variables.rs

```diff
@@ -515,15 +515,17 @@ impl Program<'_> {
             ty,
             name,
             init,
-            ..
+            meta,
         }: VarDeclaration,
     ) -> Result<Handle<Expression>, ErrorKind> {
         #[cfg(feature = "glsl-validate")]
         if let Some(ref name) = name {
             if ctx.lookup_local_var_current_scope(name).is_some() {
-                return Err(ErrorKind::VariableAlreadyDeclared(name.clone()));
+                return Err(ErrorKind::VariableAlreadyDeclared(meta, name.clone()));
             }
         }
+        #[cfg(not(feature = "glsl-validate"))]
+        let _ = meta;
```

Comment:
nit: maybe a allow unused is better here

Reply:
Please apply it only to the variable in question and not the whole struct

Reply:
I tried, but the warning still exists when only applied to the one variable.

Reply:
Oh ok then let's leave it as his